
The following have been reloaded with a version change:
  1) CUDA/9.0.176 => CUDA/9.0.176-intel-2017.u2-GCC-6.2.0-CUDA9
  2) intel/2017.u2 => intel/2017.u2-GCC-6.2.0-CUDA9


The following have been reloaded with a version change:
  1) CUDA/9.0.176-intel-2017.u2-GCC-6.2.0-CUDA9 => CUDA/9.0.176
  2) Tensorflow/1.10.0-intel-2017.u2-GCC-6.2.0-CUDA9-Python-3.5.2-GPU => Tensorflow/1.9.0-intel-2017.u2-GCC-6.2.0-CUDA9-Python-3.5.2-GPU


The following have been reloaded with a version change:
  1) GCC/6.2.0 => GCC/4.9.2
  2) GMP/6.1.1-intel-2017.u2 => GMP/6.1.1-intel-2016.u3
  3) Python/3.5.2-intel-2017.u2-GCC-6.2.0-CUDA9 => Python/2.7.11-intel-2016.u3
  4) SQLite/3.13.0-intel-2017.u2 => SQLite/3.13.0-intel-2016.u3
  5) Tcl/8.6.5-intel-2017.u2 => Tcl/8.6.4-intel-2016.u3
  6) Tk/8.6.5-intel-2017.u2 => Tk/8.6.4-intel-2016.u3-no-X11
  7) bzip2/1.0.6-intel-2017.u2 => bzip2/1.0.6-intel-2016.u3
  8) icc/2017.u2-GCC-6.2.0 => icc/2016.u3-GCC-4.9.2
  9) iccifort/2017.u2-GCC-6.2.0 => iccifort/2016.u3-GCC-4.9.2
 10) ifort/2017.u2-GCC-6.2.0 => ifort/2016.u3-GCC-4.9.2
 11) iimpi/2017.u2-GCC-6.2.0 => iimpi/2016.u3-GCC-4.9.2
 12) imkl/2017.2.174-iimpi-2017.u2-GCC-6.2.0 => imkl/11.3.3.210-iimpi-2016.u3-GCC-4.9.2
 13) impi/2017.2.174-iccifort-2017.u2-GCC-6.2.0 => impi/5.1.3.223-iccifort-2016.u3-GCC-4.9.2
 14) intel/2017.u2-GCC-6.2.0-CUDA9 => intel/2016.u3
 15) libffi/3.2.1-intel-2017.u2 => libffi/3.2.1-intel-2016.u3
 16) libreadline/6.3-intel-2017.u2 => libreadline/6.3-intel-2016.u3
 17) ncurses/6.0-intel-2017.u2 => ncurses/5.9-intel-2016.u3
 18) zlib/1.2.8-intel-2017.u2 => zlib/1.2.8-intel-2016.u3

Traceback (most recent call last):
  File "job_dataset_config.py", line 3, in <module>
    from misc_fun import FLAGS
  File "/home/oxyoung/ai_mecha/misc_fun.py", line 4, in <module>
    import tensorflow as tf
  File "/usr/local/easybuild/software/Tensorflow/1.9.0-intel-2017.u2-GCC-6.2.0-CUDA9-Python-3.5.2-GPU/lib/python3.5/site-packages/tensorflow/__init__.py", line 22, in <module>
    from tensorflow.python import pywrap_tensorflow  # pylint: disable=unused-import
  File "/usr/local/easybuild/software/Tensorflow/1.9.0-intel-2017.u2-GCC-6.2.0-CUDA9-Python-3.5.2-GPU/lib/python3.5/site-packages/tensorflow/python/__init__.py", line 49, in <module>
    from tensorflow.python import pywrap_tensorflow
  File "/usr/local/easybuild/software/Tensorflow/1.9.0-intel-2017.u2-GCC-6.2.0-CUDA9-Python-3.5.2-GPU/lib/python3.5/site-packages/tensorflow/python/pywrap_tensorflow.py", line 74, in <module>
    raise ImportError(msg)
ImportError: Traceback (most recent call last):
  File "/usr/local/easybuild/software/Tensorflow/1.9.0-intel-2017.u2-GCC-6.2.0-CUDA9-Python-3.5.2-GPU/lib/python3.5/site-packages/tensorflow/python/pywrap_tensorflow.py", line 58, in <module>
    from tensorflow.python.pywrap_tensorflow_internal import *
  File "/usr/local/easybuild/software/Tensorflow/1.9.0-intel-2017.u2-GCC-6.2.0-CUDA9-Python-3.5.2-GPU/lib/python3.5/site-packages/tensorflow/python/pywrap_tensorflow_internal.py", line 28, in <module>
    _pywrap_tensorflow_internal = swig_import_helper()
  File "/usr/local/easybuild/software/Tensorflow/1.9.0-intel-2017.u2-GCC-6.2.0-CUDA9-Python-3.5.2-GPU/lib/python3.5/site-packages/tensorflow/python/pywrap_tensorflow_internal.py", line 24, in swig_import_helper
    _mod = imp.load_module('_pywrap_tensorflow_internal', fp, pathname, description)
ImportError: /usr/local/easybuild/software/Tensorflow/1.9.0-intel-2017.u2-GCC-6.2.0-CUDA9-Python-3.5.2-GPU/lib/python3.5/site-packages/tensorflow/python/_pywrap_tensorflow_internal.so: undefined symbol: _Py_FalseStruct


Failed to load the native TensorFlow runtime.

See https://www.tensorflow.org/install/install_sources#common_installation_problems

for some common reasons and solutions.  Include the entire stack trace
above this error message when asking for help.
